statsmodels/tsa/statespace/structural.py

```diff
@@ -203,6 +214,39 @@ class UnobservedComponents(MLEModel):
     time series is available in the results class in the `seasonal`
     attribute.
 
+    ** Freq. Seasonal**
+
+    Each frequency-domain seasonal component is modeled as:
+
+    .. math::
```

Comment:
I don't think this is all formatted correctly (.. math:: will wrap in in \begin{split} \end{split} - is that what you were looking for?)